src/components/InlineCodeBlock/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import type {TText} from 'react-native-render-html';
+import Text from '@components/Text';
+import type InlineCodeBlockProps from './types';
+
+function InlineCodeBlock<TComponent extends TText>({TDefaultRenderer, textStyle, defaultRendererProps, boxModelStyle}: InlineCodeBlockProps<TComponent>) {
+    const {textDecorationLine, ...textStyles} = textStyle;
+
+    return (
+        <TDefaultRenderer
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...defaultRendererProps}
+        >
+            <Text style={{...boxModelStyle, ...textStyles}}>{defaultRendererProps.tnode.data}</Text>
+        </TDefaultRenderer>
+    );
+}
```

Comment:
```suggestion
function InlineCodeBlock<TComponent extends TText>({TDefaultRenderer, textStyle, defaultRendererProps, boxModelStyle}: InlineCodeBlockProps<TComponent>) {
    const textStyles = StyleSheet.flatten(textStyle);
    const {textDecorationLine, ...restTextStyles} = textStyles;

    return (
        <TDefaultRenderer
            // eslint-disable-next-line react/jsx-props-no-spreading
            {...defaultRendererProps}
        >
            <Text style={[boxModelStyle as StyleProp<TextStyle>, restTextStyles]}>{defaultRendererProps.tnode.data}</Text>
        </TDefaultRenderer>
    );
}
```

In order to be able to have `StyleProps` in `InlineCodeBlock` props, I suggest using `StyleSheet.flatten` to extract the style into a object (since we don't use `StyleSheet.create` in the project this won't affect performance I think), and then we assert `boxModelStyle` to get rid of the error there, also should be okay since in web `Text` can use view properties. WDYT?